js/admin.js

```diff
@@ -37,7 +39,11 @@
 						$( '.bfa-ajax-response-holder' ).html( response ).slideDown().delay(2000).fadeTo(600, 0).delay(300).slideUp().fadeTo(0, 100);
 					});
 				}
-			);
+			).fail( function(response ) {
+				$( '.bfa-loading-gif' ).fadeOut( function() {
+					$( '.bfa-ajax-response-holder' ).html( response.responseText ).slideDown().delay(2000).fadeTo(600, 0).delay(300).slideUp().fadeTo(0, 100);
```

Comment:
@MickeyKay Now I'm showing the red box instead of the ugly browser alert as you suggested.